simple_ado/__init__.py

```diff
@@ -108,6 +108,56 @@ def verify_access(self) -> bool:
 
         return True
 
+    def create_pull_request(
+        self,
+        source_branch: str,
+        target_branch: str,
+        *,
+        title: Optional[str] = None,
+        description: Optional[str] = None,
+        reviewer_ids: List[str] = [],
+    ) -> ADOPullRequestClient:
+        """Creates a pull request with the given information
+
+        :param source_branch: The source branch of the pull request
+        :param target_branch: The target branch of the pull request
+        :param title: The title of the pull request
+        :param description: The description of the pull request
+        :param reviewer_ids: The reviewer IDs to be added to the pull request
+
+        :returns: A new ADOPullRequestClient for the pull request created
+
+        :raises ADOException: If we fail to create the pull request
+        """
+        self.log.debug("Creating pull request")
+
+        request_url = f"{self.http_client.base_url()}/git/repositories/{self._context.repository_id}"
+        request_url += "/pullRequests?api-version=5.1"
+
+        body: Dict[str, Any] = {
+            "sourceRefName": _cleanup_branch_name(source_branch),
+            "targetRefName": _cleanup_branch_name(target_branch),
+        }
+
+        if title is not None:
+            body["title"] = title
+
+        if description is not None:
+            body["description"] = description
+
+        if len(reviewer_ids) > 0:
+            body["reviewers"] = [{"id": reviewer_id} for reviewer_id in reviewer_ids]
+
+        response = self.http_client.post(request_url, json_data=body)
+        ado_response = self.http_client.decode_response(response)
```

Comment:
You can just do a return here. It's up to consumers how they want to handle errors.